### ISSUE TITLE
[question] how to add translation context in v3

### ISSUE THREAD

**Issue by vonovak** [lingui/js-lingui#797]
There have been several issues opened that are related to this but they are old and closed, so opening a new one for v3. I'd like to add translator context to the comment in the po files

motivation: when I have the following code

```
import { t } from '@lingui/macro';
t`Russian`
```

when translating into Czech for example, the translator does not know if this is `Ruský` (just an adjective) or `Ruština` (Russian language).

I'd like to influence the po file output as in:

```
# * I want to be able to add a comment here please *
#: js/file/path.js:125
msgid "Russian"
msgstr ""
```

What is the current way to add such information with the `t` macro?

Found this old [commit](https://github.com/lingui/js-lingui/commit/7e3cbc3e457905128704ca7c0cbd8ef16eb31311#diff-014131efceb6c1b6215d38fa9069c989197ba99934226313fa8e4b0f46c7b325) that describes a way that probably used to work in v2. And some [rfc](https://github.com/lingui/js-lingui/issues/197#issuecomment-414688135).

Or should we use `defineMessage`? Does it work the same as the `t` macro? (it seems to return an object, not a string)

Thank you! 

I'll open a PR for the docs once the "official way" is confirmed.

**Comment by tricoder42** [comment]
Yeah, at the moment you should use `defineMessage`:

```
import { defineMessage } form "@lingui/macro"
import { i18n } from "@lingui/core"

const msg = defineMessage({
  id: "message id",
  message: "Russian"
})

i18n._(msg)
```

It should be possible (and fairly straightforward) to extend `t` macro with this syntax:

```
// not implemented, yet
t({
  id: "message id",
  message: "Russian"
})
```

... but I don't have enough time to work on this. Feel free to send a PR if you have time to dig in.

**Comment by tricoder42** [comment]
Closed in favor of #800
